Models/SSBTaxAssessment/GraphQLTypes/TaxAssessmentDatasetType.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Backend.Models.Base.JsonStat;
+using Backend.Models.SSBTaxAssessment.POCO;
+using Backend.utils;
+using GraphQL;
+using GraphQL.Types;
+
+namespace Backend.Models.SSBTaxAssessment.GraphQLTypes
+{
+    public class
+        TaxAssessmentDatasetType : JsonStatDatasetType<TaxAssessment.TaxAssessmentDimension, TaxAssessmentDimensionType>
+    {
+        public TaxAssessmentDatasetType() : base(true)
+        {
+            Field<ListGraphType<LabeledValueTaxType>>("value", "The values for the selected municipalities and years.",
+                null,
+                context =>
+                {
+                    var municipalities = context.Parent.GetArgument<List<string>>("municipalities");
+                    var years = context.Parent.GetArgument<List<string>>("years");
+                    var skipSize = context.Source.Dimension.Size[3];
+                    var municipalityEntrySize = context.Source.Dimension.Size[0];
+                    return (from municipality in municipalities
+                        let municipalityYears = context.Source.Value
+                            .Skip(NorwayTools.MunicipalityCodeToIndexTaxes[municipality] * 294)
```

Comment:
Assign the [magic number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) to a variable which describes what it is.

Reply:
Caaaaaan do! It's context.Source.Dimension.Size[2] * context.Source.Dimension.Size[3], (how many rows there are in total for each municapility)

Reply:
Superb

Reply:
Could the `294` be moved to its own variable?